lib/vdsm/storage/blockVolume.py

```diff
@@ -366,7 +365,6 @@ def calculate_volume_alloc_size(
                                               "volume_utilization_chunk_mb")
                 alloc_size = chunk_size_mb * MiB
         else:
-            # Preallocated qcow2
```

Comment:
How is this related to preallocation on file volumes? If this is unrelated to the change, best submit this as separate pr to avoid confusing reviewers.